docs/resources/logdna_view.md

```diff
@@ -73,50 +73,50 @@ resource "logdna_view" "my_view" {
 
 The following arguments are supported:
 
-_Note:_ At least one of the following properties: `query`, `host`, `app`, `tag`, `level` must be specified to create a view.
+_Note:_ At least one of the following properties: `query`, `host`, `app`, `tag`, `level` must be specified to create a View.
 
-- `name`: (Required) The name this view will be given
-- `query`: (Optional) The search query scope for the view
-- `category`: (Optional) An array of existing category names this view should be nested under. Not case sensitive. Note: if the category does not exist- the view will by default be created in uncategorized
-- `hosts`: (Optional) Array of names of hosts to filter the view by
-- `apps`: (Optional) Array of names of apps to filter the view by
-- `levels`: (Optional) Array of names of levels to filter the view by
-- `tags`: (Optional) Array of names of tags to filter the view by
+- `name`: (Required) The name this View will be given
+- `query`: (Optional) The search query scope for the View
+- `categories`: (Optional) An array of existing category names this View should be nested under. Not case sensitive. Note: if the category does not exist- the View will by default be created in uncategorized
+- `hosts`: (Optional) Array of names of hosts to filter the View by
+- `apps`: (Optional) Array of names of apps to filter the View by
+- `levels`: (Optional) Array of names of levels to filter the View by
+- `tags`: (Optional) Array of names of tags to filter the View by
```

Comment:
all are case-sensitive

Reply:
where would I want to mention that @nimbinatus?

Reply:
Ah, so `categories` is the only one that isn't. I'd add it to line 75 as part of the note like this:
```markdown
_Note:_ At least one of the following properties must be specified to create a View: `apps`, `hosts`, `levels`, `query`, `tags`. Unless otherwise noted, all values are case sensitive.
```

Reply:
The field name is case-sensitive (i.e. name cannot be NAME, query cannot be QUERY) but the values for a given field are case-insensitive (i.e. the value of name could be "Test" or "test")